README.md

```diff
@@ -8,44 +8,57 @@ This repository contains the source code for the automation tests for the Salesf
 
 ### WDIO VSCode Service
 
-This project is based on `WDIO VSCode Service, available at https://github.com/webdriverio-community/wdio-vscode-service
+This project is based on WDIO VSCode Service, available at https://github.com/webdriverio-community/wdio-vscode-service
 
 ### Getting Started
 
 If you are interested in contributing, please take a look at the [CONTRIBUTING](CONTRIBUTING.md) guide.
 
-After cloning this repo, you will also need to clone https://github.com/forcedotcom/salesforcedx-vscode. The default topography is for `salesforcedx-vscode` and `salesforcedx-vscode-automation-tests` to reside side-by-side in the same parent location.
+After cloning this repo, you will also need to have a folder called `salesforcedx-vscode` residing side-by-side in the same parent location, and have the vsixes you want to test in `salesforcedx-vscode/extensions` directory. e.g:
 
-To install the dependencies, run `npm install`. You do not need to compile - when running the e2e automation tests the code is dynamically compiled.
+```
+.
+├── ...
+├── salesforcedx-vscode-automation-tests    # E2E Tests repo
+├── salesforcedx-vscode
+│   └── extensions                          # Directory containing the salesforce extensions
+│       ├── salesforcedx-vscode-core-59.0.0.vsix
+│       ├── salesforcedx-vscode-apex-59.0.0.vsix
+│       └── ...
+└── ...
+```
 
-For the VSCode extension (`{location}/salesforcedx-vscode`), you will need to run `npm install` but you do not need to compile or perform any other steps.
```

Comment:
This line is important because the automation tests do not run unless you run `npm install` in salesforcedx-vscode.  Need to keep this.

Reply:
That's not true, remember we test against vsixes now 🙂

Reply:
Sorry I forgot running `npm install` is not necessary for testing against VSIXs 🤦‍♀️ Thanks for the reminder!